ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml

```diff
@@ -1296,17 +1225,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-knative-serverless-operator-main-4.12-vsphere-e2e-vsphere-ocp-412-continuous
-  reporter_config:
```

Comment:
Why is this reporter config deleted everywhere?

Reply:
Fixed.